console-app-dotnetcore/Program.cs

```diff
@@ -59,8 +51,7 @@ static void Main(string[] args)
 
         private static void Log(string format, params object[] args)
         {
-            _logger.LogInformation("{0}: {1}", DateTime.Now, string.Format(format, args));
-
+            Console.WriteLine("{0}: {1}", DateTime.Now, string.Format(format, args));
```

Comment:
How come?

Reply:
Removed logger initialization in the console app